src/libraries/DIRC/DDIRCLut.cc

```diff
@@ -68,16 +72,27 @@ bool DDIRCLut::brun(JEventLoop *loop) {
         loop->Get(locDIRCGeometry);
         dDIRCGeometry = locDIRCGeometry[0];
 
-	// rotation angles for bar boxes (reverse sign from survey)
-	for(int i=0; i<12; i++) {
-	  dRotationX[i] = -0.2134*TMath::DegToRad(); // +pitch // -0.00219;
-	  dRotationY[i] =  0.0963*TMath::DegToRad(); // -yaw   //  0.00181
-	  dRotationZ[i] = -0.0355*TMath::DegToRad(); // -roll  // -0.00062;
+	// get cherenkov angle corrections from CCDB
+	vector <double> thetac_offsets_bar(5184);
+	if(loop->GetCalib("/DIRC/thetac_offsets_bar", thetac_offsets_bar)) 
+	  jout << "Can't find requested /DIRC/thetac_offsets_bar in CCDB for this run!" << endl;
+	
+	for(int ibar=0; ibar<48; ibar++) {
```

Comment:
Would be nice to define these numbers in DDIRCGeometry and get their values from there.  See for example the constant factors in DFCALGeometry.